documentation/modules/eventing/pages/_partials/watching-logs.adoc

```diff
@@ -21,7 +21,7 @@ stern -n {kube-ns} -c user-container {kube-svc}
 ----
 ====
 
-The logs will have the output like below printing every minute.
+The logs will have the output like below printing every 2 minutes.
```

Comment:
All the pingsources used have a cron expression that executes in 2 mins. Fixed the docs everywhere I found.